migrator/migrations/policycategorymigrationhelper/category_migrator.go

```diff
@@ -0,0 +1,53 @@
+package policycategorymigrationhelper
+
+import (
+	"context"
+	"embed"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/jsonutil"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+const (
+	policyCategoryParentDirName = "categories_add_and_remove"
+	addDirName                  = policyCategoryParentDirName + "/add"
+	removeDirName               = policyCategoryParentDirName + "/remove"
+)
+
+// ReadPolicyCategoryFromFile reads categories from file given the path and the collection of files.
+func ReadPolicyCategoryFromFile(fs embed.FS, filePath string) (*storage.PolicyCategory, error) {
+	contents, err := fs.ReadFile(filePath)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to read file %s", filePath)
+	}
+	var category storage.PolicyCategory
+	err = jsonutil.JSONBytesToProto(contents, &category)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to unmarshal policy json at path %s", filePath)
+	}
+	return &category, nil
+}
+
+// AddNewCategoriesToStore adds new default categories to the store
+func AddNewCategoriesToStore(fs embed.FS, upsertCategory func(ctx context.Context, category *storage.PolicyCategory) error) error {
```

Comment:
going to put a nit here.  The word `Store` has loaded meaning within ACS.  To some not paying attention it may imply you are using one of the "stores" for migration and not gorm.  But this will use whatever you want to use func you pass in.  So I would probably call it `AddNewCategoriesToDatabase` or something generic like that.